### ISSUE TITLE
Prefix verbosity modes

### ISSUE THREAD

**Issue by devnoname120** [psxdev/debugnet#3]
The following macros are often already used. Would it be possible to prefix them with something like `DBN`?

```C
#define NONE 0
#define INFO 1
#define ERROR 2
#define DEBUG 3	
```
https://github.com/psxdev/debugnet/blob/69ed404b5eb83891f00ca4aac35ba6b7cdd185ca/libdebugnet/include/debugnet.h#L11-L14

**Comment by psxdev** [comment]
Yes i will update it with your request to make all compliant with new sdk/toolchain
